catroid/src/main/res/values-ar/strings.xml

```diff
@@ -326,12 +326,12 @@
   <!-- Project activity view -->
   <string name="project_name">البرنامج:</string>
   <string name="my_projects_activity_title">البرامج</string>
-  <string name="number_of_objects">Objects:</string>
+  <string name="number_of_objects">Objects: %1$d</string>
   <string name="start_scene_name">%1$s (First Scene)</string>
-  <string name="number_of_scripts">النص:</string>
-  <string name="number_of_bricks">اللّبنات:</string>
-  <string name="number_of_looks">المشهد:</string>
-  <string name="number_of_sounds">الأصوات:</string>
+  <string name="number_of_scripts">النص: %1$d</string>
+  <string name="number_of_bricks">اللّبنات: %1$d</string>
+  <string name="number_of_looks">المشهد: %1$d</string>
+  <string name="number_of_sounds">الأصوات: %1$d</string>
```

Comment:
Oh, I completely forgot... afaik these changes will be overwritten by Crowdin anyway.
You always only need to change the English strings.

@thmq, how should we handle such "translations"?
The string itself hasn't changed. However it has to be retranslated again in every language :disappointed:.